src/components/app/TabBar.css

```diff
@@ -74,8 +77,9 @@
 
 .tabBarTab.selected {
   z-index: 1;
+  padding: 6px 4px;
   border-color: var(--grey-30);
-  margin-bottom: -1px; /* Expand 1px towards the bottom to cover the tab bar bottom border. */
+  margin: 0 -0.5px; /* Expand 1px towards the bottom to cover the tab bar bottom border. */
```

Comment:
New comment:

`No bottom margin, so that this tab covers the .Details-top-bar bottom border.`